releases/release-1.13/release-notes-draft.md

```diff
@@ -422,4 +422,34 @@ SIG Windows focused on improving reliability for Windows and Kubernetes support
 - Handle Windows named pipes in host mounts. ([#69484](https://github.com/kubernetes/kubernetes/pull/69484), [@ddebroy](https://github.com/ddebroy))
 - Fixed inconsistency in windows kernel proxy when updating HNS policy. ([#68923](https://github.com/kubernetes/kubernetes/pull/68923), [@delulu](https://github.com/delulu))
 
-
+##  External Dependencies
+
+- Default etcd server is unchanged at v3.2.24 since Kubernetes 1.12. ([#68318](https://github.com/kubernetes/kubernetes/pull/68318))
```

Comment:
Should we mention the etcd client update?

Reply:
<img width="558" alt="screen shot 2018-12-03 at 1 17 39 pm" src="https://user-images.githubusercontent.com/927168/49399179-d5449300-f6fd-11e8-91ae-677fbdc844f5.png">